src/plugins/discover/public/plugin.ts

```diff
@@ -386,20 +371,17 @@ export class DiscoverPlugin
         },
       },
       ui: {
-        defaults: {},
-        reducer: () => ({}),
+        defaults: async () => {
+          this.initializeServices?.();
```

Comment:
why do we want/need the nullish coalescing here?

Reply:
This is undefined in the plugin class but defined in the plugin start function. This should in theory never be null since it will always run after the start method, but ive added this to avoid typescript complaining, and i also did not want to mark it as non null.